lldb/source/Core/Mangled.cpp

```diff
@@ -56,9 +56,9 @@ static inline bool cstring_is_mangled(llvm::StringRef s) {
 // pointers
 static ThreadSafeDenseMap<const char *, ConstString> *
 GetDisplayDemangledNamesCache() {
-  ThreadSafeDenseMap<const char *, ConstString> *g_cache;
-  std::once_flag g_flag;
-  std::call_once(g_flag, [&g_cache]() -> void {
+  static ThreadSafeDenseMap<const char *, ConstString> *g_cache = nullptr;
```

Comment:
Could we replace this whole function body with something like:
```
static ThreadSafeDenseMap<const char *, ConstString> g_cache;
return &g_cache;
```
?

Reply:
Yes. 
I retested the impact manually and it's equivalent to the initially suggested implementation.